app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -1011,10 +1011,10 @@ class SmokeTest {
         }.expandCollection(collectionName) {
             clickCollectionThreeDotButton()
             selectDeleteCollection()
-            confirmDeleteCollection()

```

Comment:
Should this move below as well? Or is `verifySnackbarText` doing that for us?

Reply:
Yes, `confirmDeleteCollection()` works with delete dialogue.  `verifySnakbarTest` works with the newly added snackbar.